manifest.md

```diff
@@ -133,16 +143,94 @@ Unlike the [image index](image-index.md), which contains information about a set
 
 ## Example of a SCRATCH config or layer descriptor
 
-Notice that the `mediaType` is subject to the usage or context, while the digest is specifically defined as `ScratchDigestSHA256`
+Notice that the `mediaType` is subject to the usage or context, while the digest is specifically defined as `ScratchDigestSHA256`.
+When the `ScratchDigestSHA256` is used, the media type SHOULD be set to `application/vnd.oci.scratch.v1+json` to differentiate the descriptor from one pointing to content.
 
 ```json,title=SCRATCH%20config&mediatype=application/vnd.oci.descriptor.v1%2Bjson
 {
-  "mediaType": "application/vnd.oci.example+json",
+  "mediaType": "application/vnd.oci.scratch.v1+json",
   "size": 2,
   "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a"
 }
 ```
 
+## Guidelines for Artifact Usage
+
+Content other than OCI container images MAY be packaged using the image manifest.
+When this is done, the `config.mediaType` value MUST be set to a value specific to the artifact type or the [scratch value](#example-of-a-scratch-config-or-layer-descriptor).
+If the `config.mediaType` is set to the scratch value, the `artifactType` MUST be defined.
+If the artifact does not need layers, a single layer SHOULD be included with a non-zero size.
```

Comment:
I am not sure if it was part of the discussion I missed (but even if so, I think the explanation should stand here alone): why would I choose to use scratch+`artifactType`, rather than just setting `config.mediaType` to whatever specific value I want to use?

Reply:
I expect any artifact producing implementation that doesn't need or want a config will use a scratch manifest.

Suppose you just want to upload an SBOM, or a cat gif (`image/gif`). What do you put in config.mediaType?

If you chose `image/gif`, some implementations might try to render the config as a gif, or save it as a .gif file on download, etc. The config descriptor however won't point to a gif blob, it'll point to `{}` and the first layer will be the `image/gif`.

Reply:
@jonboulle the issue that was raised is the `config.mediaType` must reference the content of the `config.digest` blob according to the descriptor spec. By using that for artifacts that do not have a dedicated config blob, we have a mismatch between the media type of the blob and the artifactType of the artifact.

Reply:
Sorry, I might have asked the question backwards :)
In what circumstance would I set `config.mediaType` to something other than scratch or an actual (``application/vnd.oci.image.config.v1+json`) config? Is there a use case there we really need to support?

Reply:
Thanks, that's the exact reference that was missing for me. I will think about it for another moment; I would rather it's captured more explicitly here